src/pages/recreation/gene.tsx

```diff
@@ -1,38 +1,42 @@
 import Head from 'next/head';
 import styled from 'styled-components';
-import { Page, EventCard } from 'modules';
+import { Page, Header } from 'modules';
 import { Spaces } from 'theme';
 import { Typography, Card, FluidContainer, Button } from 'components';
 
-const HeaderContent = styled.div`
-  width: 80%;
-  padding: 72px;
+const WelcomeContentContainer = styled.div`
+  text-align: center;
 `;
-
-const cards = [
-  {
-    title: 'Inclusive',
-    children:
-      'A welcoming, inclusive space to safely talk about your dietary patterns and nutritional needs.',
-    iconSrc: '/vectors/recreation/cooking.svg',
-    iconAlt: 'cooking image',
-  },
-  {
-    title: 'Community',
-    children:
-      'Empowering conversations that focus on your strengths and areas for growth.',
-    iconSrc: '/vectors/recreation/group-workout.svg',
-    iconAlt: 'community workout image',
-  },
-  {
-    title: 'Set Goals',
-    children:
-      'Goal setting, problem-solving, and ongoing accountability to reach health goals that are important to you.',
-    iconSrc: '/vectors/recreation/goal.svg',
-    iconAlt: 'goal image',
-  },
-];
 export default function Gene() {
+  const cards = [
```

Comment:
fully static content should be outside of the component

Reply:
moved cards and buttons content outside of component